ru/mail/chrome/messenger/messengercompose/pbMacroSearchOverlay.dtd

```diff
@@ -1,9 +1,9 @@
-<!ENTITY pbMacroSearchPanel.width "600">
+﻿<!ENTITY pbMacroSearchPanel.width "600">
 
 <!ENTITY insertMacro.key "l">
-<!ENTITY insertMacro.modifiers "accel">
+<!ENTITY insertMacro.modifiers "ускоритель">
 
-<!ENTITY insertMacro.label "Quick Bar">
+<!ENTITY insertMacro.label "Панель быстрого доступа">
 
-<!ENTITY pbMacroSearchPanel.firstrunWin "When typing a URL, Postbox will autocomplete against the URLs from your emails. To disable URL autocompletion, go to Options > Composition > Advanced">
-<!ENTITY pbMacroSearchPanel.firstrunMac "When typing a URL, Postbox will autocomplete against the URLs from your emails. To disable URL autocompletion, go to Preferences > Composition > Advanced">
+<!ENTITY pbMacroSearchPanel.firstrunWin "При вводе URL-адреса приложение Postbox будет автоматически дополнять его на основе адресов URL, найденных в сообщениях электронной почты. Чтобы отключить функцию автодополнения URL, выберите «Настройки > Создание > Дополнительные»">
+<!ENTITY pbMacroSearchPanel.firstrunMac "При вводе URL-адреса приложение Postbox будет автоматически дополнять его на основе адресов URL, найденных в сообщениях электронной почты. Чтобы отключить функцию автодополнения URL, выберите «Настройки > Создание > Дополнительные»">
```

Comment:
CORRECTIONS NEEDED for line 8 and 9: 

+<!ENTITY pbMacroSearchPanel.firstrunMac "При вводе URL-адреса приложение Postbox будет автоматически дополнять его на основе адресов URL, найденных в сообщениях электронной почты. Чтобы отключить функцию автодополнения URL, выберите «Настройки» > «Создание» > «Дополнительные»">

Let's surround each UI element with quotation marks, since they are related, yet separate.